myst_nb/__init__.py

```diff
@@ -35,6 +36,11 @@ def update_togglebutton_classes(app, config):
         config.togglebutton_selector += f", {selector}"
 
 
+def init_glue_cache(app):
+    if not hasattr(app.env, "glue_data"):
+        app.env.glue_data = {}
```

Comment:
It would be nice to have 'external' access to this dict, to see what keys we have stored. Maybe we can write a small function in myst-nb, that `ExecutableBookProject/cli` could utilise (outside of sphinx) to show the currently stored keys:

```python
from pathlib import Path
import pickle

def get_glue_data(doctreedir):
    path = Path(doctreedir, "environment.pickle")
    env = pickle.loads(path.read_bytes())
    return env.glue_data
```

Reply:
Or alternatively you store this as a separate JSON file in the `doctreesdir`.